cpp/src/jit/parser.cpp

```diff
@@ -53,7 +53,7 @@ std::string escape_percent(const std::string& src) {
   }
 }
 
-std::string get_rid_of_nonalnum_sqrbra(const std::string& src) {
+std::string ptx_parser::get_rid_of_nonalnum_sqrbra(const std::string& src) {
```

Comment:
Human-readable names, please. :) Is `sqrbra` a necessary part of the name? I can't tell because this function is not documented and the code is hard to understand.  Please also document it.

```suggestion
std::string ptx_parser::remove_nonalphanumeric(const std::string& src) {
```